provisioning/plugins/corim-psa-decoder/extractor.go

```diff
@@ -27,26 +46,37 @@ func (o Extractor) SwCompExtractor(rv comid.ReferenceValue) ([]*proto.Endorsemen
 	// identified in the subject of the "reference value" triple.  A single
 	// reference-triple-record SHALL completely describe the updatable PSA RoT.
 	swComponents := make([]*proto.Endorsement, 0, len(rv.Measurements))
-
+	var swComponent *proto.Endorsement
```

Comment:
swComponent is not a good name: this is now a more generic ref-val

Reply:
Corrected with a new name. This needs a further tidy up in the proto interface. Will raise a github issue for that!